src/test/java/seedu/address/logic/LogicManagerTest.java

```diff
@@ -71,14 +72,22 @@ public void execute_commandExecutionError_throwsCommandException() {
     }
 
     @Test
-    public void execute_validCommand_success() throws Exception {
+    public void execute_validCommand_MainWindow_success() throws Exception {
```

Comment:
```suggestion
    public void execute_validCommand_mainWindow_success() throws Exception {
```
I think the CI build is failing because this does not follow the test case name format